recipe/meta.yaml

```diff
@@ -19,6 +17,7 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - extension-helpers
```

Comment:
Hmm, still getting `extension-helpers is not detected` error. Is this not the right place to put this dependency?

https://github.com/conda-forge/extension-helpers-feedstock